client/my-sites/earn/components/add-edit-plan-modal/index.tsx

```diff
@@ -47,11 +47,6 @@ type DefaultNames = {
  * Return the minimum transaction amount for a currency.
  * If the defaultCurrency is not the same as the current currency, return the double, in order to prevent issues with Stripe minimum amounts
  * See https://wp.me/p81Rsd-1hN
- *
- * @param {StripeMinimumCurrencyAmounts} currency_min - Minimum transaction amount for each currency.
- * @param {string} currency - Currency.
- * @param {string} connectedAccountDefaultCurrency - Default currency of the current account
- * @returns {number} Minimum transaction amount for given currency.
```

Comment:
The types above were redundant give they're defined in the TS notation just below.